auto_round/autoround.py

```diff
@@ -857,8 +859,8 @@ def remove_duplicates(lst):
                         format = "auto_round:auto_awq"
                 elif is_nv_fp(self.data_type) or is_mx_fp(self.data_type):
                     format = f"auto_round:{self.data_type}"
-                elif is_wfp8afp8(self):  # staic wfp8afp8
-                    format = "auto_round:fp8"
+                elif is_static_wfp8afp8(self):  # staic wfp8afp8
```

Comment:
@WeiweiZhang1 you have an AR to refine formats related code, please be aware of this change